### ISSUE TITLE
COAR Notify vs. CUSAP

### ISSUE THREAD

**Issue by joschirr** [coar-notify/notify.coar-repositories.org#24]
There is a new initiative from an STM task force called CUSAP - Content-update Signaling and Alerting Protocol (CUSAP), https://www.stm-assoc.org/standards-technology/cusap/

Although the intended workflows and use cases between COAR Notify and CUSAP are somewhat different, I wonder if there are any connections between the two and if the CUSAP objective could be another use case for COAR Notify?

**Comment by martinklein0815** [comment]
Yes, IMO, the CUSAP objective could be a use case for Notify. In fact, we are in conversation with Craig and other members of the group about just that and, from what I understand, Notify is on their shortlist of technologies to consider for a future/possible implementation of their project.
